vaadin-details-flow/src/main/java/com/vaadin/flow/component/details/Details.java

```diff
@@ -38,6 +38,32 @@
     public Details() {
     }
 
+    /**
+     * Initializes a new Details using the provided summary and content.
+     *
+     * @param summary the summary text to set.
```

Comment:
Missing second param in javadoc. Also, would it make sense to add a constructor in which the content is also a String?

Reply:
Missing param added to Javadoc.
I would consider avoiding a NxN explosion of constructors. What do you feel?

Reply:
Yes, and probably content as a component is more useful.